cpp/tests/utilities/column_utilities.hpp

```diff
@@ -90,26 +99,14 @@ void print(cudf::column_view const& col, std:: ostream &os = std::cout, std::str
 template <typename T>
 std::pair<std::vector<T>, std::vector<bitmask_type>> to_host(column_view c) {
   std::vector<T> host_data;
-  std::vector<bitmask_type> host_bitmask;
-
-  auto col = column(c);
 
-  if (col.size() > 0) {
-    host_data.resize(col.size());
-    CUDA_TRY(cudaMemcpy(host_data.data(), col.view().head<T>(),
-                        col.size() * sizeof(T),
-                        cudaMemcpyDeviceToHost));
-  }
-
-  if (col.nullable()) {
-    size_t mask_allocation_bytes = bitmask_allocation_size_bytes(c.size());
-    host_bitmask.resize(mask_allocation_bytes);
-    size_t num_mask_elements = num_bitmask_words(c.size());
-    CUDA_TRY(cudaMemcpy(host_bitmask.data(), col.view().null_mask(),
-                        num_mask_elements * sizeof(bitmask_type),
+  if (c.size() > 0) {
+    host_data.resize(c.size());
+    CUDA_TRY(cudaMemcpy(host_data.data(), c.head<T>(), c.size() * sizeof(T),
                         cudaMemcpyDeviceToHost));
   }
 
+  std::vector<bitmask_type> host_bitmask = bitmask_to_host(c);
   return std::make_pair(host_data, host_bitmask);
```

Comment:
```suggestion
  return { host_data, host_bitmask };
```

Reply:
With your suggestion, the reader has to go find the return type to know what this code is doing.

Reply:
I might agree for longer functions, but the return type is only 10 lines above. However, I don't mind either way - it's just a preference.